DOC-2735 - Refresh of VM Migration Assistant documentation (#10396)
spectrocloud/librarium: docs/docs-content/vm-management/vm-migration-assistant/vm-migration-assistant.md

<!-- prettier-ignore-end -->

It provides a user-friendly interface to manage the migration process, including creating source providers, defining
migration plans, and monitoring the status of migrations.

## Limitations

> These limitations apply generally to the VM Migration Assistant.

Reviewer: ```suggestion
```

Not sure we need this?

Author: I wasn't sure how to write this. We have limitations for some of the procedures separately, but I wanted to convey that these apply "globally" but without using that word.

Reviewer: Hm, I would think since it's on the landing page that people know it's a limitation of the feature itself, but perhaps that's not a safe assumption to make.

"Following are general limitations of the VM Migration Assistant"?

Author: ```suggestion
The following are general limitations of the VM Migration Assistant.
```
